app/scripts/components/scenariobuilder.js

```diff
@@ -198,8 +191,13 @@ var exports = (function() {
             // set the template, somehow a computed setter was not working...
             this.selectTemplate(template);
 
+            // set the dataloaded to true, such that the <form> DOM will be rendered
             this.dataLoaded = true;
 
+            // Initialize the tooltips: We do this after the DOM update.
+            this.$nextTick(function () {
+              this.updateAfterTick();
+            });
```

Comment:
This has been moved from r.227: this location guarantees that the tagsinput plugin is initialized at the right time, even when cloning.